projects/angular/src/data/datagrid/datagrid-filter.ts

```diff
@@ -72,6 +73,7 @@ import { KeyNavigationGridController } from './utils/key-navigation-grid.control
       [attr.aria-label]="commonStrings.keys.datagridFilterDialogAriaLabel"
     >
       <div class="datagrid-filter-close-wrapper">
+        <span class="datagrid-filter-title">{{ filterTitle }}</span>
```

Comment:
Screen reader users might have difficulties with this not marked up as a heading. I'm not sure how to handle the heading level though.

There is an `aria-label` on the popup element, so maybe this is fine. I just want to make sure.

Reply:
The popup `aria-label` doesn't have the column name. Maybe that should be changed.

Reply:
The filter trigger already have `aria-label` similar to the filter title: `{COLUMN} filter`. That is already announced to the user via screen reader.
Adding the title ahve more value for the regular user. The title will be read by the screen reader anyway.
First will be `popup aria-label` and then the title, if I'm not mistaken.